Update limits-specifications-teams.md (#6073)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/limits-specifications-teams.md

|Number of channels per team    | 200 (includes deleted channels)&sup3;         |
|Number of Private channels per team    |30| (includes deleted channels)&sup3;
|Number of members in a Private channel    |250|

> |Maximum size of distribution list, security group or Office 365 group that can be imported in to a team    |3,500|

Reviewer: ```suggestion
|Maximum size of a distribution list, security group, or Office 365 group that can be imported into a team    |3,500|
```
